topics/galaxy-project/tutorials/outputs/tutorial.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: tutorial_hands_on
+
+title: "4. Galaxy project outputs"
```

Comment:
Consider not numbering tutorials. Instead you can use the `priority: ` field to assign an ordering/priority within the topic listing

Reply:
Excellent field to have - thanks.